agimus_controller/agimus_controller/ocp/ocp_croco_goal_reaching.py

```diff
@@ -64,7 +64,7 @@ def create_running_model_list(self) -> list[crocoddyl.ActionModelAbstract]:
                 running_cost_model,
             )
             running_model = crocoddyl.IntegratedActionModelEuler(
-                running_DAM,
+                running_DAM, stepTime=self.dt
```

Comment:
Note for later: when using non-constant step size, this piece of code needs to be updated.